### DISCUSSION TITLE
Use transformRequest with maplibre-react-components

### DISCUSSION THREAD

**Discussion by sylvain258** [maplibre/maplibre-gl-js#5270] [Q&A]
Hello,

I'm developing a React application. I'm looking at maplibre-react-components, but there doesn't seem to be a “transform request” property. I need this property to allow access to an API.  What to do it ?

Thanks

**Comment by HarelM** [comment]
You should probably open a feature request there or open a PR to add it there...

**Reply by sylvain258** [reply]
I have contacted the developer directl. Thanks
